sectioning.asciidoc

```diff
@@ -1,13 +1,19 @@
-The projects preferred markup format is http://asciidoctor.org[Asciidoc].  
-Asciidoc has a lot of great features, but denoting chapter and title information is the first step in making beautiful ebooks.
+Sectioning
+----------
 
+The project's preferred markup format is http://asciidoctor.org[Asciidoc].  
+Asciidoc has a lot of great features, but denoting title and chapter information is the first step in making beautiful ebooks.
 
-The title of the book should have a row of dashes underneath.  This turns into a Level one header +<h1>+
+== Titles
+The title of the book should have a row of hyphens underneath.  This turns it into a level-one header (+<h1>+ in HTML).
```

Comment:
While this is the recommended format in the python Asciidoc documentation.  I am now in favor of the Asciidoctor style guide, which says to use one leading `=`.

Thank you for updating this, it is very clearly out of date.